civo/datasource_kubernetes_cluster.go

```diff
@@ -64,8 +64,9 @@ func dataSourceKubernetesCluster() *schema.Resource {
 				Description: "The cni for the k3s to install (the default is `flannel`) valid options are `cilium` or `flannel`",
 			},
 			"tags": {
-				Type:        schema.TypeString,
+				Type:        schema.TypeSet,
```

Comment:
Does Set mean a slice of string?

Reply:
I use the typeSet and them I say the Elem is `&schema.Schema{Type: schema.TypeString},`